src/applications/vaos/project-cheetah/components/SecondDosePage.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { connect } from 'react-redux';
+import * as actions from '../redux/actions';
+import FormButtons from '../../components/FormButtons';
+import { scrollAndFocus } from '../../utils/scrollAndFocus';
+import moment from 'moment';
+import { getReviewPage } from '../redux/selectors';
+import AdditionalInfo from '@department-of-veterans-affairs/component-library/AdditionalInfo';
+
+const pageKey = 'secondDosePage';
+const pageTitle = 'Plan your second dose';
+
+function SecondDosePage({
+  data,
+  facility,
+  pageChangeInProgress,
+  routeToNextAppointmentPage,
+  routeToPreviousAppointmentPage,
+}) {
+  const history = useHistory();
+  const { date1, vaFacility } = data;
+
+  useEffect(() => {
+    if (history && !vaFacility) {
```

Comment:
You can remove this if statement and the replace call below

Reply:
Done